recipes/bitserializer/all/conanfile.py

```diff
@@ -50,17 +52,29 @@ def _compilers_minimum_version(self):
             "apple-clang": "12",
         }
 
-    @property
-    def _is_header_only(self):
+    def _is_header_only(self, info=False):
+        if Version(self.version) < "0.50":
+            return True
         # All components of library are header-only except csv-archive
-        return not self.options.with_csv
+        options = self.info.options if info else self.options
+        return not options.with_csv
 
-    def _patch_sources(self):
-        # Remove 'ryml' subdirectory from #include
-        replace_in_file(self, os.path.join(self.source_folder, "include/bitserializer/rapidyaml_archive.h"), "#include <ryml/", "#include <", strict=False)
+    def config_options(self):
+        if self.settings.os == "Windows" or Version(self.version) < "0.50":
```

Comment:
I think that there is need to check if current build is "header-only" like below:
```suggestion
        if self.settings.os == "Windows" or self._is_header_only():
```
BitSerializer doesn't produce any static libraries without option `with_csv`.

Reply:
It doesn't really matter, patch is valid regardless of with_csv option value.

Reply:
Good point, but you can't rely on options values in config_options, so it must be handled in configure(), I'll push a fix.